<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Minus</class>
 <widget class="QWidget" name="Minus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像相减</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_bg">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开背景图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bg_obj">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开含目标对象图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_obj">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>相减得目标对象图</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>298</width>
      <height>298</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="showPic1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>298</width>
      <height>298</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="showPic2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>298</width>
      <height>298</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="showPic3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
